Ker 209 hds 3 header

timwessman opened the pull request:
This PR replaces the old Header component with HDS 3.0.0 Header component.
[KER-209](https://helsinkisolutionoffice.atlassian.net/browse/KER-209)

How to test:
- Check that the new Header looks OK. It should have a generic HDS style
- Try change languages
- Try to login/logout
- Try mobile screen size and open the navigation menu

[KER-209]: https://helsinkisolutionoffice.atlassian.net/browse/KER-209?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

mikkojamG commented on src/components/Header/Header.jsx:
```diff
@@ -0,0 +1,147 @@
+/* eslint-disable react/no-multi-comp */
```
This ignore must have been necessary at some point, but looks like it's not needed anymore 🤔

timwessman commented on src/components/Header/Header.jsx:
```diff
@@ -0,0 +1,147 @@
+import config from "../../config";
+import { localizedNotifyError } from '../../utils/notify';
+
+class Header extends React.Component {
```
Sure. Fixed

timwessman commented on src/components/Header/Header.jsx:
```diff
@@ -0,0 +1,147 @@
+    if (newLanguage !== this.props.language) {
+      const languageParam = `lang=${newLanguage}`;
+      let searchParams;
+      if (window.location.search.includes('lang=')) {
```
Fixed this to use `URL API`'s `URLSearchParams`

timwessman commented:
Fixed all small changes.
Removed some unused css. Turku probably used some of these old navigation's css classes to customize their header, but as the component is totally changed, they will need to fix their customization also.

mikkojamG reviewed (approved):
LGTM 👍
